npm/react/src/createMount.ts

```diff
@@ -115,6 +97,23 @@ export const makeMountFn = (
       // and letting hooks and component lifecycle methods to execute mount
       // https://github.com/bahmutov/cypress-react-unit-test/issues/200
       .wait(0, { log: false })
+      .then(() => {
+        Cypress.log({
```

Comment:
Does this need to be surrounded by `if (options.log !== false) {` like the code before it was moved?

Reply:
Fixed in [acfa4c4](https://github.com/cypress-io/cypress/pull/24543/commits/acfa4c4a7cce0b73d9df22148e4640015cfe636c), was a regression fro develop.